Common/DtaDevEnterprise.cpp

```diff
@@ -747,15 +854,15 @@ uint8_t DtaDevEnterprise::setLockingRange_SUM(uint8_t lockingrange, uint8_t lock
 		LOG(D1) << "Entering DtaDevEnterprise::setLockingRange_SUM()";
         LOG(E) << "setLockingRange_SUM not implemented";
         LOG(D1) << "Exiting DtaDevEnterprise::setLockingRange_SUM()";
-	return 0;
+	return DTAERROR_INVALID_PARAMETER;
 }
 uint8_t DtaDevEnterprise::enableUser(char * password, char * userid)
 {
 	LOG(D1) << "Entering DtaDevEnterprise::enableUser";
 	LOG(E) << "enableUser not implemented";
 	if (!password && !userid) { LOG(E) << "Formal Parameters"; }
 	LOG(D1) << "Exiting DtaDevEnterprise::enableUser()";
-	return 0xff;
+	return DTAERROR_INVALID_PARAMETER;
```

Comment:
I'm confused, shouldn't this be DTAERROR_UNSUPORTED_LOCKING_RANGE?

I've seen some more such inconsistencies below, I'm afraid not to introduce some subtle bugs.